Tools/Driver.PackOne.targets

```diff
@@ -143,9 +143,14 @@ Related features:
   </PropertyGroup>
 
   <!-- $(PackOne_SourcePath) and $(PackOne_DestinationPath) -->
-  <PropertyGroup>
-    <PackOne_SourcePath>objfre_$(TargetOS)_$(TargetArch)\$(PackOne_SourcePathArch)</PackOne_SourcePath>
-    <PackOne_DestinationPath>$(PackOne_DestinationPrefix)Install\$(TargetOS)\$(TargetArch)</PackOne_DestinationPath>
+  <PropertyGroup Condition="$(Configuration.EndsWith('Debug'))"> 
+    <PackOne_SourcePath Condition="$(PackOne_SourcePath)==''">objchk_$(TargetOS)_$(TargetArch)\$(PackOne_SourcePathArch)</PackOne_SourcePath>
```

Comment:
Is this missing single quotes around the variable? I don't think it's optional, both side are expected to expand to two single quotes if the variable is empty.

`  Condition="'$(PackOne_SourcePath)'==''"`

Reply:
Yes it's missing and is really optional, how mentioned in https://msdn.microsoft.com/en-us/library/7szfhaft.aspx: "Single quotes are not required for simple alphanumeric strings or boolean values. However, single quotes are required for empty values." And it works for variables too, but actually is not matched to code style, thus I'll add quotes.

Reply:
Got it, thanks!